xbmc/video/VideoDatabase.cpp

```diff
@@ -1385,6 +1409,48 @@ int CVideoDatabase::AddToTable(const std::string& table, const std::string& firs
   return -1;
 }
 
+int CVideoDatabase::AddRatings(int mediaId, const char *mediaType, const RatingMap& values, const std::string& default)
+{
+  int ratingid = -1;
+  try
+  {
+    if (NULL == m_pDB.get()) return -1;
+    if (NULL == m_pDS.get()) return -1;
+    if (values.size() > 0)
```

Comment:
Please always use curly braces around multi-line blocks. But you don't really need the `values.size() > 0` check (which should be `!values.empty()`) because the loop won't do anything on an empty map.